src/pages/billing/List/List.ts

```diff
@@ -82,14 +84,14 @@ export const List = ({ queryParams: { selectedName } }) => {
   )(() => {
     if (Auth.hasBillingScope()) {
       return Ajax(signal).Billing.listAccounts()
-        .then(_.keyBy('accountName'))
+        .then(_.keyBy('accountName')) // @ts-ignore
```

Comment:
Perhaps someone can help me figure out how to avoid this ignore (note that `listAccounts` is not TS).